api/src/pcapi/core/educational/api/dms.py

```diff
@@ -6,16 +6,17 @@
 from pcapi.connectors.dms import api as dms_api
 from pcapi.connectors.dms import models as dms_models
 from pcapi.core.educational import models as educational_models
-from pcapi.core.offerers import models as offerers_models
 from pcapi.models import db
 
 
 logger = logging.getLogger(__name__)
 
 
-def import_dms_applications(procedure_number: int) -> None:
+def import_dms_applications(procedure_number: int, ignore_previous: bool = False) -> None:
     """import dms applications for eac status"""
-    previous_import = _get_previous_import(procedure_number)
+    previous_import = None
+    if ignore_previous:
+        previous_import = _get_previous_import(procedure_number)
```

Comment:
Donc si `ignore_previous` est à `True`, on va chercher le previous ? Il y a un nom à changer, ou un `not` à rajouter